indico/modules/events/management/controllers/cloning.py

```diff
@@ -101,21 +100,32 @@ class PatternCloneCalculator(CloneCalculator):
     def _calculate(self, form):
         args = {'dtstart': self._naivify(form.start_dt.data)}
         args.update(self._calc_stop_criteria(form))
-        return self._tzify(rrule.rrule(rrule.MONTHLY,
-                                       interval=form.num_months.data,
-                                       byweekday=form.week_day.week_day_data,
-                                       bysetpos=form.week_day.day_number_data,
-                                       **args))
+        return (self._tzify(rrule.rrule(rrule.MONTHLY,
```

Comment:
This will be more readable if you write it like this:

    dates = self._tzify(...)
    return dates, false